### PR TITLE
PWN-3074 - Fix crash in reset seed phrase screen

### PR DESCRIPTION
**Pull request by kuznetsov**

### CODE REVIEW

File: app/src/main/java/org/p2p/wallet/settings/ui/reset/seedphrase/ResetSeedPhraseFragment.kt [hunk 1]
```diff
@@ -84,7 +85,11 @@ class ResetSeedPhraseFragment :
                 keysRecyclerView.isVisible = true
                 phraseAdapter.addSecretKey(SecretKey())
             }
-            messageTextView.text = buildSeedInfoText()
+
+            messageTextView.apply {
```
Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/settings/ui/reset/seedphrase/ResetSeedPhraseFragment.kt:89]: scope function is used when there are more than 2 calls

File: app/src/main/java/org/p2p/wallet/settings/ui/reset/seedphrase/ResetSeedPhraseFragment.kt [hunk 2]
```diff
@@ -111,7 +116,14 @@ class ResetSeedPhraseFragment :
     }
 
     private fun buildSeedInfoText(): SpannableString {
-        val message = getString(R.string.auth_recover_info)
+        val seedInfo = getString(R.string.settings_what_is_a_security_key)
+
+        val message = StringBuilder().apply {
```
Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/settings/ui/reset/seedphrase/ResetSeedPhraseFragment.kt:121]: can be replaced with `buildString`

**Review by 1sakov — APPROVED** [review]
Approved from my side.
Didn't get what was the reason of the crash? 
It didn't find the span?

**Review by kabramovp2p — APPROVED** [review]

**Comment by kuznetsov** [comment]
> Approved from my side. Didn't get what was the reason of the crash? It didn't find the span?

Spanned text was missing in general message, that led to exception in `span.setSpan()`

**Pull request merged**